src/scenes/RoadieRunScene.jsx

```
@@ -13,11 +14,14 @@ export const RoadieRunScene = () => {
 
   const controllerFactory = useMemo(() => createRoadieStageController, [])
 
-  const logic = useMemo(() => ({
-    gameStateRef,
-    stats,
-    update
-  }), [gameStateRef, update])
+  const logic = useMemo(
```

Biome (2.4.4)

[error] This hook does not specify its dependency on stats.

(lint/correctness/useExhaustiveDependencies)